app/config/parameters.yml.dist

```diff
@@ -53,3 +53,6 @@ parameters:
 
     applicolis_api_key: ~
     applicolis_api_base_url: ~
+
+    # For the guzzle client
+    osrm_url: 'http://osrm:5000'
```

Comment:
I add an other parameter because I was not sure if the osmr_host parameter is actually used somewhere else. 
But I am agree to remove the new one and just replace the old. I fix it